docs/install.rst

```diff
@@ -20,4 +20,7 @@ The repository uses `git-lfs (large file storage) <https://git-lfs.github.com>`_
     conda env update --file environment.yml
     pip install -e .
 
-
+Related Publication
```

Comment:
Having this in `install` is a bit odd, but to be honest there's not really a great place to put it either (as you mentioned in this week's meeting). How would we feel about having it in "getting started" instead? Or is that too late on in the process?

Also should we have a link to the preprint in the "contributions" section too? (i.e. in addition to having it elsewhere)

Reply:
The getting started page is a notebook, so that's why I didn't put it there but I do think that is the best spot, so I will move it there.